test/spec/modules/33acrossAnalyticsAdapter_spec.js

```diff
@@ -372,7 +394,7 @@ describe('33acrossAnalyticsAdapter:', function () {
           this.enableAnalytics();
```

Comment:
I think the title of this unit test is confusing.. 
```
 context('and the incomplete report has been sent successfully', function () {
        it('it sends a report string', function () {
```

maybe change it to something that mentions what's in the assertion.